themes/uhsg_theme/uhsg_theme.theme

```diff
@@ -419,11 +419,6 @@ function uhsg_theme_preprocess_node(&$variables) {
   if (in_array($bundle, ['article', 'theme']) && $variables['page']) {
     _uhsg_theme_add_link_to_instructions_for_students($variables);
   }
-  if ($bundle == 'article') {
-    $variables['new_students_label'] = t('New students');
-    $variables['new_students_description'] = t('New students study in the new degree programmes that started August 1, 2017. Students studying according to the old degree structure are referred to as old students.');
```

Comment:
As these terms aren't used anywhere else, we can remove these from translation files too.

Reply:
Removed unused translations.